kuma/core/helpers.py

```diff
@@ -150,8 +151,11 @@ def entity_decode(str):
 
 
 @register.function
-def inlinei18n(locale):
-    return StaticI18nJob().get(locale)
+def statici18n(locale):
+    if not locale:
+        locale = settings.LANGUAGE_CODE
+    filename = get_filename(locale, settings.STATICI18N_DOMAIN)
+    return static(os.path.join(settings.STATICI18N_OUTPUT_DIR, filename))
```

Comment:
On second thought, we may not need to use Pipeline at all. If we use `staticfiles_storage.url` here and add *locale* to `STATICFILES_DIRS`, collectstatic will version of the filename and the asset will be served up with a far-future Expires header. I haven't worked out all the kinks yet, but that should work.